CONTRIBUTING.md

```diff
@@ -1,6 +1,7 @@
 # Contributing
 
 Welcome to the Ankaios community. Start here for info on how to contribute and help improve our project.
```

Comment:
```suggestion
Welcome to the Ankaios community. Start here for info on how to contribute and help to improve our project.
```

Reply:
I would leave it as it is

Reply:
I have asked the technical writer of Eclipse SDV for a recommendation.

Reply:
"help improve", but either is fine.

Reply:
So let's keep it as it is.